lib/hammer_cli_foreman/role.rb

```diff
@@ -6,25 +6,28 @@ class Role < HammerCLIForeman::Command
 
     resource :roles
 
+    module RoleDataExtender
+      def extend_data(data)
+        data['builtin_text'] = (data['builtin'].to_i > 0) ? _('Yes') : _('No')
+        data
+      end
+    end
+
     class ListCommand < HammerCLIForeman::ListCommand
+      include RoleDataExtender
+
       output do
         field :id, _("Id")
         field :name, _("Name")
+        field :builtin_text, _("Builtin")
```

Comment:
I fixed boolean formatter in hammer-cli: https://github.com/theforeman/hammer-cli/pull/219
Would you mind using
```ruby
field :builtin, _("Builtin"), Fields::Boolean
```
instead to keep commands consistent? It should work fine with the fix above.